PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -452,253 +517,312 @@ log_Msg "POSIXLY_CORRECT="${POSIXLY_CORRECT}
+  # (applied after GDC, which we don't bundle in, because of the possible need
+  # to average multiple T1/T2 inputs).
+
+  # This overwrites the {T1w,T2w}_acpc_dc outputs created above, and mimics what
+  # occurs at the beginning of PostFreeSurfer/CreateMyelinMaps.sh.
+  # Note that the CreateMyelinMaps equivalent is still needed though because
+  # (1) T1w_acpc_dc_restore_brain is (re)generated with a better estimate of
+  #     the brain mask, provided by FreeSurfer
+  # (2) the entire set of T2w_acpc_dc outputs needs to be regenerated, using the
+  #     refinement to the "T2wtoT1w" registration that FreeSurfer provides.
+
+  # Just implement inline, rather than writing a separate script
+  # Added 2/19/2019
+  # ------------------------------------------------------------------------------
+
+  log_Msg "Creating one-step resampled version of {T1w,T2w}_acpc_dc outputs"
+
+  # T1w
+  OutputOrigT1wToT1w=OrigT1w2T1w_PreFS  # Name for one-step resample warpfield
+  convertwarp --relout --rel --ref=${T1wTemplate} --premat=${T1wFolder}/xfms/acpc.mat --warp1=${T1wFolder}/xfms/${T1wImage}_dc --out=${T1wFolder}/xfms/${OutputOrigT1wToT1w}
+
+  OutputT1wImage=${T1wFolder}/${T1wImage}_acpc_dc
+  applywarp --rel --interp=spline -i ${T1wFolder}/${T1wImage} -r ${T1wTemplate} -w ${T1wFolder}/xfms/${OutputOrigT1wToT1w} -o ${OutputT1wImage}
+  fslmaths ${OutputT1wImage} -abs ${OutputT1wImage} -odt float  # Use -abs (rather than '-thr 0') to avoid introducing zeros
+  fslmaths ${OutputT1wImage} -div ${T1wFolder}/BiasField_acpc_dc ${OutputT1wImage}_restore
+  fslmaths ${OutputT1wImage}_restore -mas ${T1wFolder}/${T1wImage}_acpc_dc_brain ${OutputT1wImage}_restore_brain
+
+  if [ ! "${T2wInputImages}" = "NONE" ] ; then
+    OutputOrigT2wToT1w=OrigT2w2T1w_PreFS  # Name for one-step resample warpfield
+    convertwarp --relout --rel --ref=${T1wTemplate} --premat=${T2wFolder}/xfms/acpc.mat --warp1=${T1wFolder}/xfms/${T2wImage}_reg_dc --out=${T1wFolder}/xfms/${OutputOrigT2wToT1w}
+
+    OutputT2wImage=${T1wFolder}/${T2wImage}_acpc_dc
+    applywarp --rel --interp=spline -i ${T2wFolder}/${T2wImage} -r ${T1wTemplate} -w ${T1wFolder}/xfms/${OutputOrigT2wToT1w} -o ${OutputT2wImage}
+    fslmaths ${OutputT2wImage} -abs ${OutputT2wImage} -odt float  # Use -abs (rather than '-thr 0') to avoid introducing zeros
+    fslmaths ${OutputT2wImage} -div ${T1wFolder}/BiasField_acpc_dc ${OutputT2wImage}_restore
+    fslmaths ${OutputT2wImage}_restore -mas ${T1wFolder}/${T1wImage}_acpc_dc_brain ${OutputT2wImage}_restore_brain
+
+    # Remove the file (warpfield) that serves as a proxy in FreeSurferPipeline for whether PostFreeSurfer has been run
+    # i.e., whether the T1w/T2w_acpc_dc* volumes reflect the PreFreeSurferPipeline versions (above)
+    # or the PostFreeSurferPipeline versions
+    OutputOrigT2wToT1wPostFS=OrigT2w2T1w  #Needs to match name used in both FreeSurferPipeline and PostFreeSurferPipeline
+    imrm ${OutputOrigT2wToT1wPostFS}
+  fi
```

Comment:
Since `OrigT2w2T1w` is our new proxy for whether PostFS has been run, we need to remove L797-801, and use the following in its place _outside_ of the conditional:
```
    # Remove the file (warpfield) that serves as a proxy in FreeSurferPipeline for whether PostFreeSurfer has been run
    # i.e., whether the T1w/T1w_acpc_dc* volumes reflect the PreFreeSurferPipeline versions (above)
    # or the PostFreeSurferPipeline versions
    OutputOrigT1wToT1wPostFS=OrigT1w2T1w  #Needs to match name used in both FreeSurferPipeline and PostFreeSurferPipeline
    imrm ${OutputOrigT1wToT1wPostFS}
```

Reply:
Done.